MODULE.bazel

```diff
@@ -1,13 +1,13 @@
 module(
     name = "gazelle",
-    version = "0.37.0",
+    version = "0.38.0",
     repo_name = "bazel_gazelle",
 )
 
-bazel_dep(name = "bazel_features", version = "1.9.1")
-bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "bazel_features", version = "1.11.0")
```

Comment:
We shouldn't bump `bazel_dep` versions unless we actually need features from the new versions (keep the rules_go bump to get rid of the override)